xray/commands/audit/applicabilitymanager.go

```diff
@@ -53,7 +54,10 @@ func GetExtendedScanResults(results []services.ScanResponse, dependencyTrees []*
 		return nil, fmt.Errorf(applicabilityScanFailureMessage, err.Error())
 	}
 	if !shouldRun {
-		log.Debug("conditions to run applicability scan are not met, didnt exec analyzer manager")
+		if len(serverDetails.Url) > 0 {
+			log.Warn("To include 'Contextual Analysis' information as part of the audit output, please run the 'jf c add' command before running this command.")
+		}
+		log.Debug("conditions to run applicability scan are not met, didn't exec analyzer manager")
```

Comment:
conditions to run applicability scan are not met, didn't exec analyzer manager
-->
The conditions required for running 'Contextual Analysis' as part of the audit are not met.